src/simplay-web/src/event/MoveNearEvent.ts

```diff
@@ -19,18 +19,21 @@ export class MoveNearEvent extends Event {
       this.args.target
     );
     // Math.random() - 0.5 is used to get random position on the circumference of the circle around the center of the cell
-    const x = targetDisplayObject.x + (Math.random() - 0.5) * context.tileWidth;
+    const x =
+      targetDisplayObject.container.x +
+      (Math.random() - 0.5) * context.tileWidth;
```

Comment:
we can - I'm just not sure that this is a magic number. Magic numbers - as per my understanding - are constant values whose origins are unclear. In this case - since 0.5 is used to create a circle around a point - the origin is clear, and extracting it into a constant might make the code less readable - especially since the meaning of it is explained in the comment above.

What do you think?